pennylane/devices/default_qubit.py

```diff
@@ -747,14 +747,7 @@ def _apply_unitary_einsum(self, state, mat, wires):
         )
 
         # We now put together the indices in the notation numpy's einsum requires
-        einsum_indices = (
-            "{new_indices}{affected_indices},{state_indices}->{new_state_indices}".format(
-                affected_indices=affected_indices,
-                state_indices=state_indices,
-                new_indices=new_indices,
-                new_state_indices=new_state_indices,
-            )
-        )
+        einsum_indices = f"{new_indices}{affected_indices},{state_indices}->{new_state_indices}"
```

Comment:
From which PR does this change come from?

Reply:
Hi @rmoyard, this is something new. Codefactor was not happy here and I thought we could solve this for good. 😅 the rest of such changes were also because of that, including the dict() -> {} change.

Reply:
@antalszava Ok thank you it makes sens 💯